### PR TITLE
Adding pull request template

### PR DESCRIPTION
**Pull request by carrtim93**
Added a pull request template to assist with onboarding contributors more efficiently.
Let me know if there is something that can be added to the template.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Related Issue
+- issue goes here
```
Comment (Lawouach) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:2]: A sentence would be better, something that says:

```
Please create an issue, to discuss the problem, and link it to this PR.
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# Additional Information
+- any additional information or context
+
+# Checklist
+- [ ] Tests
+- [ ] Changelog Entry
+- [ ] Squash Commits
+- [ ] Documentation
+
+# Screenshots
```
Comment (Lawouach) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:17]: Not required. This is a command line tool, screenshots aren't usually very helpful.

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+- issue goes here
+
+# Proposed Changes
+- change 1
+- change 2
+
+# Additional Information
+- any additional information or context
+
+# Checklist
```
Comment (Lawouach) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:11]: Please add a description before the list:

```
Please ensure that you have completed the following lists before submitting your PR:
```

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+- [ ] Tests
+- [ ] Changelog Entry
+- [ ] Squash Commits
```
Comment (Lawouach) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:14]:
```
Squashed commits to simplify and speed up the review
```

**Review by Lawouach — CHANGES_REQUESTED** [review]
Thanks. A few tweaks.

**Comment by CharlieMoon37** [comment]
I have created an updated template as per the changes requested here #237

**Pull request closed without merging**